bazel/update.sh

```diff
@@ -35,7 +39,7 @@ function update_version() {
         echo "Already on disk, download not necessary"
     fi
 
-    sed -i "s/Version: .*/Version:         ${version}/" bazel.spec.spec
+    sed -i "s/Version: .*/Version:        ${version}/" bazel.spec
```

Comment:
nit: I guess we could match the spaces in a `(group)` and insert them in the replace part using `\1`, then we don't need to make sure the number of whitespaces is right (yes I've been doing too much `sed` lately)

Reply:
that's not the focus of this pr. I only changed strictly what was necessary for update.sh to succeed with my changes. Further improvements can be done in another PR.